app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.order(:name)
```

Comment:
Your intuition is correct in that this is not the most RESTful implementation - you'll want to encapsulate the ordering into a model method in the Ingredient model (a class method that does this exact ordering) and then you can call this class method here!